### PR TITLE
Improve Tool Recipes and Textures

### PR DESCRIPTION
**Pull request by TechLord22**
This PR does numerous things concerning tools.

First, it makes changes to many textures, removing the previous dullness present. It also changes the textures of the power units, which leads into the second change.

Power Units, and thus all electric tools, now use materials of their battery tier. LV tools use Steel, MV aluminium, etc.

Electric tool recipes have also changed. All of them must be made with a power unit, and the step bypassing this has been removed. The net material cost does not change, and it greatly reduces recipe clutter.

All tools that have tool heads now also require the toolhead for crafting. This was done again to prevent clutter. Three of the earlygame tools: the hard hammer, the file, and the screwdriver are still craftable with early materials. These materials are Bronze, Iron, and Wrought Iron. The screwdrivers previously did not have recipes using their tool head, and now do.

Recipe registration for the mortars and soft mallets have been moved to the proper place with the other tool recipes.

The Universal Spade has been removed, which was rather useless.

Unused textures and models for the Grafter and Plow have also been removed.

### CODE REVIEW

File: src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java [hunk 1]
```diff
@@ -77,7 +78,7 @@ public static void registerPowerUnitRecipes() {
 
                 ModHandler.addShapedEnergyTransferRecipe(recipeName, powerUnitStack,
                         Ingredient.fromStacks(batteryStack), false,
-                        "S  ", "GMG", "PBP",
+                        "S d", "GMG", "PBP",
```
Comment (htmlcsjs) [inline src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java:81]: whats the `d` for?

Comment (TechLord22) [reply src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java:81]: The `d` adds a screwdriver to the recipe, which should have been there originally, but was forgotten in GTCE.

Comment (htmlcsjs) [reply src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java:81]: fair

**Review by htmlcsjs — APPROVED** [review]

File: src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java [hunk 2]
```diff
@@ -178,22 +157,22 @@ public static void processStick(OrePrefix stickPrefix, Material material, ToolPr
...
-                    'P', new UnificationEntry(OrePrefix.plate, material),
-                    'T', new UnificationEntry(OrePrefix.screw, material));
 
-            ModHandler.addShapedRecipe(String.format("branch_cutter_%s", material.toString()),
-                    MetaItems.BRANCH_CUTTER.getStackForm(material),
-                    "PfP", "PdP", "STS",
-                    'S', new UnificationEntry(stickPrefix, material),
-                    'P', new UnificationEntry(OrePrefix.plate, material),
-                    'T', new UnificationEntry(OrePrefix.screw, material));
+            if (material.hasFlags(GENERATE_BOLT_SCREW)) {
```
Comment (DStrand1) [inline src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java:161]: does this need `GENERATE_PLATE` as well?

**Review by DStrand1 — APPROVED** [review]

File: src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java (hunk 2 above)
Comment (TechLord22) [reply src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java:161]: Plate is in the outer if statement.

**Pull request merged**